pkg/execute/kubectl/guard.go

```diff
@@ -135,6 +135,11 @@ func (g *CommandGuard) GetAllowedResourcesForVerb(verb string, allConfiguredReso
 
 // GetResourceDetails returns a Resource struct for a given resource type and verb.
 func (g *CommandGuard) GetResourceDetails(selectedVerb, resourceType string) (Resource, error) {
+	_, found := resourcelessVerbs[selectedVerb]
```

Comment:
When we know that a given verb is just resourceless instead of error we can return empty data with no error. Alternatively, I can ignore in kubectl builder logic the `ErrResourceNotFound` error. Both works for me, but from the logic perspective I found this more suitable.

Reply:
Sounds good to me. Please just add a unit test for this test case (this will be super fast as there are table tests already) to make sure we won't accidentally break the contract 👍